templates/repo/settings/options.tmpl

```diff
@@ -763,6 +756,44 @@
 						</div>
 					</div>
 				{{end}}
+
+				{{if not .Repository.IsFork}}
+					<div class="divider"></div>
```

Comment:
Why this `divider` (and below) is needed? IIRC the `flex-item` has proper `border-top: 1px solid var(--color-secondary);`.